tests/test_game_wrapper_mario.py

```diff
@@ -0,0 +1,62 @@
+#
+# License: See LICENSE.md file
+# GitHub: https://github.com/Baekalfen/PyBoy
+#
+
+import pytest
+from pyboy import PyBoy, WindowEvent
+from tests.utils import supermarioland_rom
+
+
+@pytest.mark.skipif(not supermarioland_rom, reason="ROM not present")
+def test_mario_basics():
+    pyboy = PyBoy(supermarioland_rom, window_type="dummy", game_wrapper=True)
+    pyboy.set_emulation_speed(0)
+    assert pyboy.cartridge_title() == "SUPER MARIOLAN"
+
+    mario = pyboy.game_wrapper()
+    mario.start_game(world_level=(1, 1), win_count=0)
+
+    assert mario.score == 0
+    assert mario.lives_left == 2
+    assert mario.time_left == 400
+    assert mario.world == (1, 1)
+    assert mario.fitness == 0 # A built-in fitness score for AI development
+    pyboy.stop()
+
+
+@pytest.mark.skipif(not supermarioland_rom, reason="ROM not present")
+def test_mario_advanced():
+    pyboy = PyBoy(supermarioland_rom, window_type="dummy", game_wrapper=True)
+    pyboy.set_emulation_speed(0)
+    assert pyboy.cartridge_title() == "SUPER MARIOLAN"
+
+    mario = pyboy.game_wrapper()
+    mario.start_game(world_level=(3, 2), win_count=2)
+    # assert pyboy.get_memory_value(0xFF9A) == 2 # Win count
+    lives = 99
+    mario.set_lives_left(lives)
+    pyboy.tick()
+
+    assert mario.score == 0
+    assert mario.lives_left == lives
+    assert mario.time_left == 400
+    assert mario.world == (3, 2)
+    assert mario.fitness == 10000*lives + 6510 # A built-in fitness score for AI development
```

Comment:
The rewards here are quite high, in AI for a neural network to learn we try to keep values between -1, 1 or with mean 0 and standard deviation of 1.
If rewards are that high, it might lead to some difficulties for new AI programmers.